command/agent/auth/gcp/gcp.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/hashicorp/vault/command/agent/auth"
 	"github.com/hashicorp/vault/sdk/helper/parseutil"
 	"golang.org/x/oauth2"
-	iam "google.golang.org/api/iam/v1"
+	iam "google.golang.org/api/iamcredentials/v1"
```

Comment:
Nit: Can we reference directly `iamcredentials` instead of overwriting the package name, especially since `google.golang.org/api/iam/v1` exists?

Reply:
Yeah we should, thanks. I made that change in e0172e5bd